### PR TITLE
Standardise github action

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

**Comment by hadley** [comment]
Fixes #353

File: .github/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.html
```
Comment (scottmmjackson) [inline .github/.gitignore:1]: Is this to exclude rendered output of Rmds?

**Review by scottmmjackson — CHANGES_REQUESTED** [review]
This looks good. It makes me think that maybe a `HACKING.md`/`CONTRIBUTING.md` containing instructions (i.e.: the list of environment variables you should set) for configuring a full test run would be helpful.

**Review by scottmmjackson — APPROVED** [review]

File: .github/.gitignore (hunk 1 above)
Comment (hadley) [reply .github/.gitignore:1]: Yeah, if you have any `.md` files in that directory (it's automatically added by some helper in usethis)

**Comment by hadley** [comment]
Yeah, that would be nice to do one day 😄 At least the skip messages now tell you what you need to set.

**Pull request merged**